app/serializers/oembed_serializer.rb

```diff
@@ -45,7 +45,7 @@ def html
           <div style="font-weight: 500;">View on Mastodon</div>
         </a>
       </blockquote>
-      <script src="#{full_asset_url('embed.js', skip_pipeline: true)}" async="true"></script>
+      <script data-allowed-prefixes="#{embed_short_account_status_url(object.account, object)}" src="#{full_asset_url('embed.js', skip_pipeline: true)}" async="true"></script>
```

Comment:
Probably `root_url` here?

Reply:
You could embed multiple posts from the same server though, that's even more likely than from different servers

Reply:
Yeah, but someone copying the snippet would have multiple `<script>` wouldn't they? I can change it though.

Reply:
But only one of them would execute, no? I'm not actually sure.